src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/AdjournCaseIt.java

```diff
@@ -36,6 +36,24 @@
 @AutoConfigureMockMvc
 public class AdjournCaseIt extends AbstractEventIt {
 
+    public static final String DIRECTIONS_DUE_DATE_PLACEHOLDER = "DIRECTIONS_DUE_DATE_PLACEHOLDER";
+    public static final String NOT_LISTED_STRAIGHT_AWAY_WITH_DIRECTIONS_MADE_JSON =
+        "callback/adjournCaseGeneratedVideoWhenCaseNotListedStraightAwayWithDirectionsMade.json";
+    public static final String CCD_ABOUT_TO_SUBMIT = "/ccdAboutToSubmit";
+    public static final String AN_TEST = "An Test";
+    public static final String CASE_ID = "12345656789";
+    public static final String NINO = "JT 12 34 56 D";
+    public static final String DRAFT_ADJOURNMENT_NOTICE1 = "DRAFT ADJOURNMENT NOTICE";
+    public static final String CHESTER_MAGISTRATE_S_COURT = "Chester Magistrate's Court";
+    public static final String FIRST_IN_THE_MORNING_SESSION_ON_A_DATE_TO_BE_FIXED = "It will be first in the morning session on a date to be fixed";
```

Comment:
Minor - could this name be shortened to "FIRST_MORNING_SESSION_ON_DATE_TO_BE_FIXED"? Reduces line length for readability. Otherwise, could put value on new line similar to above json path.

Reply:
ok